container/apptabs_internal_test.go

```diff
@@ -13,6 +13,9 @@ import (
 func TestAppTabs_tabButtonRenderer_SetText(t *testing.T) {
 	item := &TabItem{Text: "Test", Content: widget.NewLabel("Content")}
 	tabs := NewAppTabs(item)
+	// bug report #3980 : buttons will be rendered while already empty tabs. after fixed it, there has another logic that
```

Comment:
I don't think we should leave bug references in code once they are resolved

Reply:
well. i just want to explain why i need to add code in other test code, i wlll remove the bug reference